docs/4-entry-points/4-guardians-registry.md

```diff
@@ -20,131 +20,136 @@ This module is in the one handling all the staking/unstaking logic for the guard
     - Save the guardian token address
     - Save the total active balance limit
 
-### 4.4.2. Activate
+### 4.4.1. Stake
```

Comment:
Not sure, but `4.4.1` is taken by the constructor already, so should this be `4.4.2`?